src/components/modal/modal.js

```diff
@@ -77,6 +77,10 @@ export const props = {
     type: Boolean,
     default: false
   },
+  noSubmitOnEnter: {
+    type: Boolean,
+    default: false
+  },
```

Comment:
This also assumes people want to close modal on <kbd>ENTER</kbd>, which is currently not the default behavior and would be considered a breaking change.

Reply:
Also, `submit` assumes a form is being used...  so its probably not the best name for the prop, as modals don't "submit"